module/cpf.sh

```diff
@@ -7,6 +7,9 @@ Site's color printf module
 #. The Color PrintF Module -={
 : ${SIMBOL_IN_COLOR?}
 
+STDERR="&2"
```

Comment:
If we're going to make global variables that are this generic, then they should live in libsimbol in the appropriate section.  Furthermore I'd rename them and drop the ampersand (and use that at call-time); so `global -i FD_STDERR=2`; then `>&${FD_...}`